app/tests/dumpDataBase.page.js

```diff
@@ -0,0 +1,22 @@
+import { Selector } from 'testcafe';
+import { PAGE_IDS } from '../imports/ui/testIDs/pageIDs';
+import {COMPONENT_IDS} from "../imports/ui/testIDs/componentIDs";
+
+class DumpDataBasePage {
+  constructor() {
+    this.pageId = `#${PAGE_IDS.DUMP_DATABASE}`;
+    this.pageSelector = Selector(this.pageId);
+  }
+
+  async isDisplayed(testController) {
+    await testController.wait(30000).expect(this.pageSelector.exists).ok();
```

Comment:
Waiting 30000 milliseconds is about 30 seconds, which is pretty excessive. It will slow test suite significantly.